FRW/Services/Resources/UrlProvider.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  UrlProvider.swift
+//  FRW
+//
+//  Created by Antonio Bello on 11/20/24.
+//
+
+import Foundation
+
+enum AccountType {
+    case flow, evm
+    
+    init(isEvm: Bool) {
+        self = isEvm ? .evm : .flow
+    }
+    
+    static var current: Self {
+        return EVMAccountManager.shared.selectedAccount == nil ? .flow : .evm
+    }
+}
+
+extension FlowNetworkType {
+    func getTransactionHistoryUrl(accountType: AccountType, transactionId: String) -> URL {
+        let baseUrl = getHistoryBaseUrl(accountType: accountType)
+        return URL(string: "\(baseUrl)/tx/\(transactionId)")!
```

Comment:
Can we don't use force unwrap in here? Case transactionId is dynamic, not a static value